index.js

```diff
@@ -44,7 +47,8 @@ class Storage {/* eslint-disable max-classes-per-file */
 
 // UI Class: Displays listed Books
 
-class UI {/* eslint-disable max-classes-per-file */
+class UI {
+  /* eslint-disable max-classes-per-file */
```

Comment:
- [ ] Great job working on this milestone. You are not suppose to disable eslint, rather fix the errors resulting from running the linter checks.